src/_guides/libraries/writing-package-pages.md

```diff
@@ -0,0 +1,294 @@
+anywhere high on the page as well. 
+
+The package [`in_app_purchase`][]
+starts its README with a brief explanation of the package and a caution.
+
+{% asset libraries/package-page-description.png alt="description of the package in_app_purchase" class="screenshot" %}
+
+
+### 2. Include visual content to explain things
+
+UI is difficult to explain in text.
+If your package page is just a “wall of text” with no visual content,
+users may find it intimidating.
+In that case, it is very likely that
+package searchers skip your package without reading the README.
+Place visual content such as images, gifs, movs
+close to the beginning of the README,
+so that users can easily find them and use them.
+(Movs or gifs are even better than static images!) 
+
+See the example below to see how
+the visual contents made the README look better (right) than before (left). 
+
+{% asset libraries/package-page-example-iap.png alt="in_app_purchase readme without and with images" class="screenshot" %}
+
+{{ site.alert.tip }}
+  When adding visual contents,
+  make sure that you use absolute URLs for the files,
+  so that they'll work no matter where the README is published.
+  One place to host your images is in the repository itself.
+  For example, here's where in_app_purchases hosts its animated gifs.
+{{ site.alert.end }}
+
+If your package supports complex concepts,
+utilize diagrams that explain the package,
+so that your users can quickly understand the package and
+feel confident about using it. 
+
+### 3. Use lists to present key information
+
+Use lists to draw attention to important information on your README.
+This section suggests to utilize lists for the followings:
+
+*   Key features of the package
+*   Parameters, attributes, or properties
+*   Unusual requirements 
+*   Things that are out of scope of your package
+*   Content of longer pages or sections (just like this list)
+
+
+#### List key features of the package 
+
+First of all, clearly list what your package can do.
+Some users might be looking for a very specific feature in your package.
+Help those users find out whether your package supports their needs or not.
+You may use tables, too.
+
+{% asset libraries/package-page-features-list.png alt="list of features of the package in_app_purchase" class="screenshot" %}
+
+{% asset libraries/package-page-features-table.png alt="list of features of the package just_audio in a table format" class="screenshot-narrow" %}
+
+
+#### List parameters, attributes, or properties
+
+You can also consider listing parameters, attributes, or properties
+for their quick reference.
+See the example below.
+
+{% asset libraries/package-page-list-property.png alt="list of supported schemes of the package url_launcher" class="screenshot" %}
+
+
+Linking specific functions or classes to those in dartdoc can be useful too.
+See [async](https://pub.dev/packages/async) package for
+an example of linking dartdocs.
```

Comment:
Yikes, these links aren't working!

Reply:
@jonasfj Do you know when the new version is going to be published?

Reply:
Filed https://github.com/dart-lang/async/issues/168

Reply:
I think we just need to publish it...